Add caching for resolved dependencies (#49)
debricked/actions: README.md

The action needs one environmental variable: `DEBRICKED_TOKEN`, to be set to your Debricked API token.
You should store it in a secret variable under `Settings - Secrets` in your repository, so it doesn't leak through the logs!

> This is an example workflow file which using Docker image:

Reviewer: ```suggestion
This is an example workflow file which uses our Docker image:
```

    steps:
      - uses: actions/checkout@v3
      - uses: debricked/actions/scan@v3.0.1
        env:
          DEBRICKED_TOKEN: ${{ secrets.DEBRICKED_TOKEN }}
      - uses: debricked/actions/cache@v3.0.1
```

> But it is also possible to run it standalone:

Reviewer: ```suggestion
But it is also possible to run it standalone, making it possible for you to customise the runtime environment:
```

This command analyses your project to find eligible manifest files, that do not have related lock files, and uses them to generate the appropriate Debricked lock files.

> Example workflow using Docker image:

Reviewer: ```suggestion
Example workflow using our Docker image:
```

      - uses: debricked/actions/resolve-non-docker@v3.0.1
        env:
          DEBRICKED_TOKEN: ${{ secrets.DEBRICKED_TOKEN }}

>       - uses: debricked/actions/cache@v3.0.1

Reviewer: keep
